README.md

```diff
@@ -93,6 +93,10 @@ After a successful build, you should have a `glslc` executable somewhere under
 the `$BUILD_DIR/glslc/` directory, as well as a `libshaderc` library somewhere
 under the `$BUILD_DIR/libshaderc/` directory.
 
+The default behavior on MSVC is to link with the static CRT. If you would like
```

Comment:
Consider something like "...MSVC is to statically link all libraries and executables, obviating the need for Visual Studio redistributables on the target system".